modules/social_features/social_event/social_event.install

```diff
@@ -2163,3 +2163,35 @@ function social_event_update_11403(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Add new "Hide enrollments" field to Event node type.
+ */
+function social_event_update_11404(): void {
```

Comment:
@nechai Since this is a new feature, it should go to the next Major release - 11.5.0
That's why we need to start update hook numbers with `115xx`.